pennylane/ops/qubit/special_unitary.py

```diff
@@ -418,7 +418,7 @@ def compute_matrix(theta, num_wires):
 
             U(\theta) = \exp(i\sum_{m=1}^d \theta_m P_m)
 
-        See :func:`~.ops.qubit.matrix_ops.pauli_basis_strings` for the ordering of Pauli words.
+        See :func:`~.ops.qubit.special_unitary.pauli_basis_strings` for the ordering of Pauli words.
```

Comment:
not entirely sure what it takes for Sphinx to include this module-level function, but it doesn't appear to exist on [latest](https://docs.pennylane.ai/en/latest/code/api/pennylane.SpecialUnitary.html) or on [your PR's docs](https://xanaduai-pennylane--3651.com.readthedocs.build/en/3651/code/api/pennylane.SpecialUnitary.html). In other words, the new link is busted because it's pointing to nothing :/ could you make it visible in this PR?

Reply:
I guess I'd have to add it somewhere manually to the docs? :thinking:

Reply:
I noticed that the ordering of Pauli words also is discussed in the class docstring, so I refer to that instead of the `pauli_basis_strings` function :)

Reply:
Resolving :)